contributing.md

```diff
@@ -122,15 +122,24 @@ uses. They are located in `//src/java_tools/...`, `//src/objc_tools/...` and
 `//src/tools/...` and their directories contain README files describing their
 respective utility.
 
-When modifying Bazel, you want to make sure that the following still works:
+When modifying Bazel, you want to make sure you can compile Bazel and
+all of the tests still pass. To do so:
 
-*  Build a distribution archive with `bazel build //:bazel-distfile`. After
-   unzipping it in a new empty directory, run `bash compile.sh all` there.
-   It rebuilds Bazel with `./compile.sh`, Bazel with the
-   `compile.sh` Bazel and Bazel with the Bazel-built binary. It compares if the
-   constructed Bazel builts are identical and then runs all bazel tests with
-   `bazel test //src/... //third_party/ijar/...`. This is what we use at Google
-   to ensure that we don't break Bazel when pushing new commits, too.
+1.  Build a distribution archive with `bazel build //:bazel-distfile`.
```

Comment:
So to sumarize there is only one step:
`bazel test` then if all test passed, you are good to go, if some test fails, you might want to look at ci too see if you are the culprit or the test is already broken at head.

Reply:
So do you need to run the `bash compile.sh all` script? 

Or just compile Bazel with your changes and run `bazel test` ?

Reply:
You don't need to run bash compile.sh all (this does not work anymore actually) neither compile bazel, only bazel test (it will compile automatically bazel)